src/api/include/migraphx/migraphx.h

```diff
@@ -70,6 +70,12 @@ typedef const struct migraphx_program* const_migraphx_program_t;
 typedef struct migraphx_onnx_options* migraphx_onnx_options_t;
 typedef const struct migraphx_onnx_options* const_migraphx_onnx_options_t;
 
+typedef struct migraphx_op_names* migraphx_op_names_t;
+typedef const struct migraphx_op_names* const_migraphx_op_names_t;
```

Comment:
Can we name this `migraphx_quantize_op_names`?

Reply:
renamed